### PR TITLE
chore(website): more protractor vs vanilla wd documentation

### PR DESCRIPTION
**Pull request by sjelin**
[Review on Reviewable](https://reviewable.io/reviews/angular/protractor/2289)

### CODE REVIEW

File: docs/webdriver-vs-protractor.md [hunk 1]
```diff
@@ -27,3 +27,31 @@ tests.
 
 If you need the vanilla WebDriver `browser` object, you can access it via
 `browser.driver`
+
+Chaining WebDriver Actions
```
Comment (juliemr) [inline docs/webdriver-vs-protractor.md]: Can we send readers to http://angular.github.io/protractor/#/locators instead, and maybe add a note about chaining actions there?

Comment (sjelin) [inline docs/webdriver-vs-protractor.md]: Done.  Let me know if this is what you had in mind.

Comment (juliemr) [inline docs/webdriver-vs-protractor.md]: Let's change the text to be a bit more general:

```
Using ElementFinders
------------------------------
In Protractor, use the `element` function to find and interact with elements through an `ElementFinder` object. This extends a WebDriver `WebElement` by adding chaining and utilities for dealing with lists. See
[locators#actions](/docs/locators) for details.
```

**Comment by sjelin** [comment]
Should be good now

**Comment by juliemr** [comment]
Looks like it's missing a push, I don't see the latest changes.

**Comment by sjelin** [comment]
Sorry, I forgot to `-f`

**Pull request merged**